.eslintrc.json

```diff
@@ -6,8 +6,15 @@
     "es6": true,
     "mocha": true
   },
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
```

Comment:
@Moise1 help me understand what does this line do?

Reply:
It handles the below error message from HoundCI by specifying the ES version and sourceType

 'import' and 'export' may appear only with 'sourceType: module' errors.

Reply:
@Moise1  I thought that what line 6 ` "es6": true,` was doing, however I more interested in  the functionality of this  `"sourceType": "module",`